### PR TITLE
Add builds for Plone 4.0 and Plone 4.1

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,18 +7,23 @@ cache:
 env:
 - PLONE_VERSION=4.2
 - PLONE_VERSION=4.3
-- PLONE_VERSION=4.3 TEST_EGGS=plone.app.contenttypes
+- PLONE_VERSION=4.3 TEST_EGGS=buildout:test-eggs=plone.app.contenttypes
```
Comment (hvelarde) [inline .travis.yml]: I think we can remove this build and same some CO₂ emissions now that Plone 5 builds are passing.

Comment (mauritsvanrees) [inline .travis.yml]: Yeah, that would be fine.

**Comment by hvelarde** [comment]
@mauritsvanrees Plone 4.1 build is passing, but Plone 4.0 build is failing because of Products.MailHost version; can you help here?

**Comment by hvelarde** [comment]
@mauritsvanrees I tried but I failed; I wait for your advice.

**Comment by mauritsvanrees** [comment]
Ah, it fails because since 278215ed7c9fd51b97d2a6807366eef0743270e2 we require `plone.app.robotframework` in the tests, even though we do not use it ourselves, apparently because `plone.app.event` needs it and does not pin it. And `plone.app.robotframework` pulls in `Products.MailHost`, for which there is no correct version for Plone 4.0.

I have removed the `plone.app.robotframework` test dependency. Let's see.

Hm, possible `zc.buildout` 2.9.0 problems...

**Comment by mauritsvanrees** [comment]
Buildout problem raised [upstream](https://github.com/buildout/buildout/issues/375).

**Comment by hvelarde** [comment]
as an alternative we can try `pip install -U setuptools==33.1.1` also; I'll let you decide.

**Comment by mauritsvanrees** [comment]
Seems good like this. Thanks.

**Comment by mauritsvanrees** [comment]
I have released 1.7.1.

**Pull request merged**